bindings/matrix-sdk-ffi/src/room_member.rs

```diff
@@ -101,10 +102,153 @@ impl RoomMember {
             Ok(())
         })
     }
+
+    pub fn can_ban_users(&self) -> bool {
+        self.inner.can_do(PowerLevelAction::Ban)
+    }
+
+    pub fn can_invite_users(&self) -> bool {
+        self.inner.can_do(PowerLevelAction::Invite)
+    }
+
+    pub fn can_kick_users(&self) -> bool {
+        self.inner.can_do(PowerLevelAction::Kick)
+    }
+
+    pub fn can_redact_events(&self) -> bool {
+        self.inner.can_do(PowerLevelAction::Redact)
+    }
+
+    pub fn can_send_state_event(&self, state_event: StateEvent) -> bool {
+        self.inner.can_do(PowerLevelAction::SendState(state_event.into()))
+    }
+
+    pub fn can_send_event(&self, event: MessageLikeEvent) -> bool {
+        self.inner.can_do(PowerLevelAction::SendMessage(event.into()))
+    }
+
+    pub fn can_trigger_notification(&self, notification: NotificationPowerLevel) -> bool {
+        self.inner.can_do(PowerLevelAction::TriggerNotification(notification.into()))
+    }
```

Comment:
Why did you expose this as many functions, rather than one `can_do` function and a `PowerLevelAction` enum, like in the Rust API?

Reply:
Because I think it's a more client friendly API.
I think `PowerLevelAction` it's a bit too generic.

Reply:
Alright, convinced. As you can see below this review thread, we now have roughly the same methods in Ruma. We'll bump the Ruma version soon, then we can introduce the same methods on the main crate's `Member` type, and just forward them here.

Reply:
> then we can introduce the same methods on the main crate's Member type, and just forward them here.

What do you mean? Introduce this set of APIs for the `RoomMember` in the `matrix-idk-base` crate?

It's better for me to wait the next bump before I continue then? If so, when you think it will be done?

Reply:
Okay so now that is merged and I also added functions very similar to the FFI ones in #1855, can you rebase your PR and match up the FFI methods with the new ones in the main SDK crate?

Reply:
Sure, on it today